### DISCUSSION TITLE
Does Huawei Cloud DNS have an agenda?

### DISCUSSION THREAD

**Discussion by identify4cloud** [go-acme/lego#1826] [Q&A] [answered]
Does Huawei Cloud DNS have an agenda?

**Comment by ldez** [comment, marked as answer]
hello, you can follow this issue https://github.com/go-acme/lego/issues/1543

**Reply by identify4cloud** [reply]
Thank you for your answer. I would like to ask whether Huawei cloud DNS will be adapted in the next version or planned to be adapted in the next few versions?

**Reply by ldez** [reply]
lego is open-source, so Huawei will be implemented when someone will implemented it.
Maybe you want to create a pull request?

**Reply by identify4cloud** [reply]
Thank you for your answer. I am currently at entry level in the go language. I am trying to create a pull request.
